src/trading/getQuote.ts

```diff
@@ -181,13 +177,11 @@ export async function getQuoteRaw(
   }
 
   return {
-    isEthFlow,
-    quote,
+    ...commonResult,
     appDataInfo,
-    orderBookApi,
-    tradeParameters,
```

Comment:
Here it returns NOT original tradeParameters, where sellToken is replaced with wrapped token